CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
```

Comment:
Is there any way to avoid modifying the whole changelog doc here? Maybe there's some option in that tool

Reply:
I was just the CLion that wanted to help with formatting. Fixed.